### ISSUE TITLE
Project Images and Screenshots

### ISSUE THREAD

**Issue by TerseTears** [TerseTears/maani.nvim#1]
![maani_logo](https://user-images.githubusercontent.com/36173945/132120638-adad3a59-54eb-4a24-a6fd-62afbe5aef46.png)
![scr1](https://user-images.githubusercontent.com/36173945/132120652-2e36a149-e6c7-479a-b804-8512d482bcf2.png)
![scr2](https://user-images.githubusercontent.com/36173945/132120657-afdaf01c-6fb6-445f-bb3d-63c9351625d8.png)
![scr3](https://user-images.githubusercontent.com/36173945/132120661-27391ef7-2642-4072-8804-0366cbf773c3.png)

**Comment by TerseTears** [comment]
![Screenshot_20211103_163917](https://user-images.githubusercontent.com/36173945/140066006-6080840a-3114-4f2c-84fd-63aadaa26780.png)
![Screenshot_20211103_160811](https://user-images.githubusercontent.com/36173945/140066044-14de55c5-eab6-4bd9-9d1c-a5268e7bb8da.png)
![Screenshot_20220312_150157](https://user-images.githubusercontent.com/36173945/158016445-e57518d5-28d4-4aff-a30d-89429aafa567.png)

**Comment by TerseTears** [comment]
![Screenshot_20220907_170417](https://user-images.githubusercontent.com/36173945/188880275-b9844165-cdb8-4715-9308-4efdd9d71dd4.png)

**Comment by TerseTears** [comment]
![Screenshot_20220907_173430](https://user-images.githubusercontent.com/36173945/188885714-5d0e599b-d538-4d66-85ee-ce653456d96a.png)

**Comment by luisfrocha** [comment]
> ![Screenshot_20211103_163917](https://user-images.githubusercontent.com/36173945/140066006-6080840a-3114-4f2c-84fd-63aadaa26780.png)

Unrelated, but how are you getting the content to show through the results dialog? I've looked at Telescope, NvimTree, and NeoTree and can't find anything to set their background to transparent AND show the buffer (it just shows through the entire window). Are you using something else?

**Comment by TerseTears** [comment]
@luisfrocha For Telescope, you can add the `winblend` option to a value other than 0 in the defaults part of the config, like this:

```lua
    config = function()
      require("telescope").setup({
  	defaults = {
	    layout_strategy = 'flex',
  	    winblend=30,
       ...
```

**Comment by luisfrocha** [comment]
@TerseTears Indeed, that worked. Thank you! If only I could also get it on NeoTree! 😄

**Comment by luisfrocha** [comment]
For anyone finding this later, I found the solution by setting the option winblend and pumblend and that sets the NeoTree window.

```
vim.opt.winblend = 30
vim.opt.pumblend = 30
```
